x/kavamint/abci.go

```diff
@@ -29,8 +32,20 @@ func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
 	}
 
 	// ------------- Community Pool -------------
-	// TODO: mint tokens for community pool
-	// TODO: send tokens to community pool
+	communityPoolInflation, err := k.AccumulateCommunityPoolInflation(ctx, totalSupply, previousBlockTime)
+	if err != nil {
+		panic(err)
```

Comment:
nit: might be useful to include the totalSupply and previousBlockTime in this error message?